readme.md

```diff
@@ -8,12 +8,8 @@ Static resources are transformed with [Jekyll](http://jekyllrb.com/).
 
 Any pages with the `.md` extension are rendered as `.html` with [rdiscount](https://github.com/davidfstr/rdiscount) (Jekyll's default Markdown parser [doesn't do nested lists](https://github.com/bhollis/maruku/issues/55)).  So in addition to installing Jekyll, you'll want to install rdiscount.
 
-    gem install jekyll rdiscount
+    gem install jekyll rdiscount rogue
```

Comment:
Good catch.  `rouge` it is.  Corrected in ad534daf004947a8796c9728053e7660f95dd0e5.